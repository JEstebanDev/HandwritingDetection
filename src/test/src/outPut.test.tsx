import { render, screen } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import OutPut from "../../src/components/outPut";
import { dataTestOutPut } from "../values/dataOutPut.js";
import { vi } from "vitest";
describe("working propertly", () => {
  test("should render the OutPut", () => {
    const { container } = render(
      <MemoryRouter initialEntries={["/"]}>
        <OutPut Data={dataTestOutPut} />
      </MemoryRouter>
    );
    expect(container).toMatchSnapshot();
  });

  test("should render the text output", () => {
    const { container } = render(
      <MemoryRouter initialEntries={["/"]}>
        <OutPut Data={dataTestOutPut} />
      </MemoryRouter>
    );
    expect(screen.getByText(/somos un equipo/i));
  });
});
