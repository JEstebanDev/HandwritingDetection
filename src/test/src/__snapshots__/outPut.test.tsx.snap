// Vitest Snapshot v1

exports[`working propertly > should render the DragDrop 1`] = `
<div>
  <div
    class="listCards"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-1ri6ub7-MuiPaper-root-MuiCard-root"
    >
      <img
        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-o69gx8-MuiCardMedia-root"
        height="200"
        src="blob:http://127.0.0.1:5173/f7f25ab6-0ab0-4216-aeb4-3668a774c27f"
      />
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
        >
          Somos un
Equipo
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`working propertly > should render the OutPut 1`] = `
<div>
  <div
    class="listCards"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-1ri6ub7-MuiPaper-root-MuiCard-root"
    >
      <img
        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-o69gx8-MuiCardMedia-root"
        height="200"
        src="blob:http://127.0.0.1:5173/f7f25ab6-0ab0-4216-aeb4-3668a774c27f"
      />
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
        >
          Somos un
Equipo
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`working propertly > should render the OutPut 2`] = `
<div>
  <div
    class="listCards"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root card css-1ri6ub7-MuiPaper-root-MuiCard-root"
    >
      <img
        class="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-o69gx8-MuiCardMedia-root"
        height="200"
        src="blob:http://127.0.0.1:5173/f7f25ab6-0ab0-4216-aeb4-3668a774c27f"
      />
      <div
        class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
      >
        <div
          class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
        >
          Somos un
Equipo
        </div>
      </div>
    </div>
  </div>
</div>
`;
